### PR TITLE
Add a history navigation button

### PR DESCRIPTION
**Pull request by chrismayer**
This adds a simple history navigation button in order to interact with the HTML5 History API.

Since the History API is supported by current browsers down to IE10 the usage of this might need a polyfill (like https://github.com/browserstate/history.js/) in older browsers, see  https://caniuse.com/#search=history for details.

@terrestris/devs how do you deal with that?

### CODE REVIEW

File: src/view/button/History.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    /**
+     * Called when the button is clicked.
+     * Performs the navigation action
+     *
+     * @param {Ext.Button} btn The history button itself
+     */
+    onBtnClick: function(btn) {
+        var me = this;
+        if (me.mode === 'HISTORY') {
+            if (btn.direction === 'back') {
```
Comment (marcjansen) [inline src/view/button/History.js]: Should this be uppercase? and below as well?

File: src/view/button/History.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+/* Copyright (c) 2015-present terrestris GmbH & Co. KG
```
Comment (ahennr) [inline src/view/button/History.js]: Change to 2018 plz

**Comment by chrismayer** [comment]
Thanks for the review guys :+1: 

I put a comment about the possibly needed polyfill in the API-Docs of the class. So this should be good to go once travis gets green.

File: src/view/button/History.js (hunk 2 above)
Comment (chrismayer) [reply src/view/button/History.js]: DOne in f122b4a

File: src/view/button/History.js (hunk 1 above)
Comment (chrismayer) [reply src/view/button/History.js]: Sure, was a long day today :wink:

**Pull request merged**